### PR TITLE
Update p5.js, import from npm

### PR DESCRIPTION
**Pull request by joshlory**
Fixes an issue in p5.sound where the metronome can get out of sync when changing tabs.

Are there specific tests I should be running or manual scenarios to try when upgrading p5?  Will affect all Game Lab levels.

### CODE REVIEW

**Comment by islemaster** [comment]
+ErinP since this affects Game Lab.

File: apps/Gruntfile.js [hunk 1]
```diff
@@ -192,7 +192,7 @@ describe('entry tests', () => {
         // p5.play repo at https://github.com/code-dot-org/p5.play
```
Comment (islemaster) [inline apps/Gruntfile.js:192]: Update comment to accurately describe source of p5.js?

File: apps/package.json [hunk 2]
```diff
@@ -150,6 +150,7 @@
     "newrelic": "^1.27.2",
     "node-js-marker-clusterer": "^1.0.0",
     "node-sass": "^4.0.0",
+    "p5": "^0.7.1",
```
Comment (islemaster) [inline apps/package.json:153]: Have we also got our p5.play fork building & testing against p5 0.7.1?

**Comment by islemaster** [comment]
I think our automated testing over Game Lab behaviors has gotten a lot better, but I'd be super-paranoid about this change.  There have been repeated issues with p5 version updates being incompatible with upstream p5.play ([example](https://github.com/molleindustria/p5.play/pull/140)) and it's unclear what subset of those will apply to our product.  In particular I'd check spritesheet loading and positioning, we've seen breaking changes in the p5 APIs used there.

**Comment by joshlory** [comment]
What's the best way to test sprite sheets?  I'm having trouble running the built-in examples like https://github.com/code-dot-org/p5.play/blob/master/examples/animation_sprite_sheet.html.

**Comment by islemaster** [comment]
You should be able to run the examples using `npm start` to launch the dev server then navigating to `localhost:8080/examples`.

**Comment by joshlory** [comment]
Hmm I'm still getting console warnings for http://127.0.0.1:8080/examples/index.html?fileName=animation_sprite_sheet.js:

image


```
Warning undefined frame 0
Warning undefined frame 0
Warning undefined frame 0
```

**Comment by islemaster** [comment]
This is also an issue upstream; see https://github.com/molleindustria/p5.play/issues/136 and my proposed fix https://github.com/molleindustria/p5.play/pull/140 which I never merged because it got sidetracked by some other discussion... I should really follow up on that.

**Pull request closed without merging**